### ISSUE TITLE
Is there any way to position=insideMiddle?

### ISSUE THREAD

**Issue by jacquesuhlemann** [recharts/recharts#4469]
Hey,

i want to build a pie chart with labels outside of the pie curved around them..
there i have two problems..

1. position is only for InsideStart and InsideEnd available and there is no chance to give a spacing/width outwards (https://recharts.org/en-US/api/Label)
Is there any chance to center this? And/or give a spacing?

2. Alternatively, I could use a customized solution and use textPath, but I don't have the scope for this.
Or am i on the wrong track here?

Thank you for your support!

**Comment by jacquesuhlemann** [comment]
Hey @ckifer !

I mean something like this..

<img width="400" alt="Bildschirmfoto 2024-04-30 um 22 01 07" src="https://github.com/recharts/recharts/assets/18728221/cd6fec2e-72e5-4547-8d10-f1cc3a6b7ee7">

I know that there is position=insideStart/insideEnd but there are no more settings for the viewpoint or for a better positioning.. I also try it with a custom label but there i missing a circumference from the cell to add a textPath.

Perhaps you have an idea how I can implement this?

**Comment by ckifer** [comment]
You're correct that this isn't currently supported without custom work and even with custom work, you might not have the attributes available that you need to do it.

I'll try to get a sandbox going to see if I can make anything work

**Comment by ckifer** [comment]
Sandbox here, but I don't really know what I'm doing  😅 
https://codesandbox.io/p/sandbox/pie-chart-of-straight-angle-forked-ws5ts7

**Comment by jacquesuhlemann** [comment]
hmm then I'll keep trying..

Thanks anyway, @ckifer  🙏
